netlogo-gui/src/main/lab/gui/Supervisor.scala

```diff
@@ -2,21 +2,21 @@
 
 package org.nlogo.lab.gui
 
-import collection.mutable.ListBuffer
+import collection.mutable.{ ListBuffer }
```

Comment:
minor point - when importing one item, braces should not be used. Maybe another item came and went